### PR TITLE
Added loan types & documents

### PR DESCRIPTION
**Pull request by aleguacaran**
Added respective files
- Database migrations
- Models
- Controllers
- Views

### CODE REVIEW

File: app/controllers/loan_types_controller.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    redirect_to loan_types_path
+  end
+
+  private
+
+  def set_loan_type
+    @loan_type = LoanType.includes(:documents).find(params[:id])
+  end
+
+  def set_documents
```
Comment (luis-luna-aguilar) [inline app/controllers/loan_types_controller.rb:51]: @aleguacaran this belongs to a helper, it is not used in the controller, therefore should be a view helper.

File: app/views/documents/_form.slim [hunk 2]
```diff
@@ -0,0 +1,23 @@
+= form_for @document do |f|
+  .row.justify-content-center.mt-2
+    .col-12.col-sm-8.col-md-6
+      label.form-label Title
+      = f.text_field :title, class: 'form-control'
+
+      - if @document.errors[:title].any?
```
Comment (luis-luna-aguilar) [inline app/views/documents/_form.slim:7]: @aleguacaran a gem like simple form would've saved you the need to handle error messages manually by input.

File: app/views/layouts/_flash.slim [hunk 3]
```diff
@@ -0,0 +1,18 @@
+- flash.each do |key, value|
+  ruby:
+    puts "key", key, "value", value
```
Comment (luis-luna-aguilar) [inline app/views/layouts/_flash.slim:3]: @aleguacaran what's this for?

**Review by luis-luna-aguilar — CHANGES_REQUESTED** [review]

File: app/views/layouts/_flash.slim (hunk 3 above)
Comment (aleguacaran) [reply app/views/layouts/_flash.slim:3]: That partial displays flash messages in the layout using Bootstrap alerts

File: app/views/documents/_form.slim (hunk 2 above)
Comment (aleguacaran) [reply app/views/documents/_form.slim:7]: Yeah, i forgot it, i have worked with formtastic before

**Pull request merged**